implementation/src/main/java/io/smallrye/config/Converters.java

```diff
@@ -408,7 +408,8 @@ public T convert(final String value) {
             if (pattern.matcher(value).matches()) {
                 return delegate.convert(value);
             }
-            throw new IllegalArgumentException(String.format(ILLEGAL_ARGUMENT_EXCEPTION_FORMAT, "Value does not match pattern", pattern, value));
+            throw new IllegalArgumentException(
```

Comment:
Looks like that reformat issue snuck in here :)

Reply:
I just pushed the format to master and rebased this